sync-core/src/main/java/com/cloudant/mazha/CouchURIHelper.java

```diff
@@ -61,6 +63,17 @@ public URI changesUri(Map<String, Object> query) {
                 this.rootUriString,
                 "_changes"
         );
+
+        //lets find the since parameter
+        if(query.containsKey("since")){
```

Comment:
would it be better to do the JSON encoding at the call site (whichever method is preparing the _changes url) rather than special-casing it here?